python/orca/test/bigdl/orca/learn/ray/experimental/test_mmcv_estimator_ray_backend.py

```diff
@@ -63,6 +63,39 @@ def train_step(self, data, optimizer, **kwargs):
         return {'loss': loss}
 
 
+class Model2(nn.Module):
+
+    def __init__(self):
+        super().__init__()
+        self.fc1 = nn.Linear(50, 50)
+        self.relu1 = nn.ReLU()
+        self.dout = nn.Dropout(0.2)
+        self.fc2 = nn.Linear(50, 100)
+        self.prelu = nn.PReLU(1)
+        self.out = nn.Linear(100, 1)
+        self.out_act = nn.Sigmoid()
+        self.loss_fn = nn.BCELoss()
+
+    def forward(self, input_, labels):
+        a1 = self.fc1(input_)
+        h1 = self.relu1(a1)
+        dout = self.dout(h1)
+        a2 = self.fc2(dout)
+        h2 = self.prelu(a2)
+        a3 = self.out(h2)
+        y = self.out_act(a3)
+        loss = self.loss_fn(y, labels)
+        return loss
+
+
+def batch_processor(model, data, train_mode, **kwargs):
+    features, labels = data
+    loss = model(features, labels)
+    log_vars = dict()
+    log_vars["var1"] = 1.0
```

Comment:
how do you test with `var1`, we need to make sure your log _buff design is working.